snakemake.el

```diff
@@ -350,6 +350,17 @@ targets."
                                        (snakemake-all-rules))
              " "))
 
+(defun snakemake-graph-workflow ()
+  (interactive)
+  """Creates a graph for the current snakefile."""
+  (let ((dot (shell-command-to-string "which dot")))
```

Comment:
I'd prefer to have this as a customizable variable, as is done with `snakemake-program`.